### ISSUE TITLE
Change emojis used in the table

### ISSUE THREAD

**Issue by JanPokorny** [Mixaill/awesome-gog-galaxy#70]
OK, I might be a little pedantic here... But I think that the current set of symbols in the table is a little confusing. Namely ⬜ looks just like ✅ without the tick, thus implying that "there is a box to tick" -- while it means the complete opposite. Also, "✅" is not the "true opposite" of ❌ -- "✔️" is.

I propose:

- ✔️ implemented
- ~✔️~ in progress / works with issues (`~✔️~`)
- ❌ not implemented
- **ⁿ/ₐ** unsupported by the platform
- ~⬛ unknown / to be added~ (not used, also I don't really understand the meaning...)

Pardon my perfectionism. If you greenlight this change, I will gladly make a PR.

**Comment by UncleGoogle** [comment]
Btw
I dont think WorkInProgress as an internal dev process is worth to mention here

**Comment by JanPokorny** [comment]
@UncleGoogle I believe that the intention is to mark "beta" features that are buggy or incomplete, thus "work on progress".

**Comment by UncleGoogle** [comment]
yup, I know, it's just I would call it "beta/works with issues" as it should reflect state of a feature. "Work in progress" is rather state of someones task/PR that shouldn't be merged to master branch until done. And it implies that someone is actively working on that feature. I have not strong feelings about wording here - your issue is about changing icons and I like it.

I'm only not sure about `~✔️~` is it how it will be rendered and displayed? Those icons/emoji look differs depending on OS. On Windows with Chrome they're look like that:

![00ad5262a373fa5ff06ed24f5d62a3cd](https://user-images.githubusercontent.com/12197825/73185187-e6c8d580-411d-11ea-9f1a-0ea9faa345d7.png)

**Comment by JanPokorny** [comment]
@UncleGoogle Interesting, are you on Windows 10? I'm on Windows 10 and it uses the OS emoji set for me in both Chrome and Firefox, including "boxy" ✅ and crossable ✔️.

This is how the post looks for me:

---
![obrazek](https://user-images.githubusercontent.com/4580066/73186760-98690600-4120-11ea-949b-8280ab048c1a.png)
---

If `~✔️~` does not render correctly on some systems, it's out of game. Then I'd just stick with "⚠".

It might also be beneficial to add explanation for each "⚠" by linking an appropriate issue/readme and writing a summary in the hover text, like so: ![⚠](#70 "Only lists installed games").

**Comment by UncleGoogle** [comment]
Heh, its Win7 that got EOL recently. IMO its OK to ignore it.
However on Firefox it looks ok:
[![Screenshot from Gyazo](https://gyazo.com/cb5ff57dacad22eefec24f032cb87b31/raw)](https://gyazo.com/cb5ff57dacad22eefec24f032cb87b31)

 But we need to check macOS.

> It might also be beneficial to add explanation for each "⚠" by linking an appropriate issue/readme and writing a summary in the hover text, like so: ⚠.

I like it.

**Comment by JD557** [comment]
On Mac OS

Firefox:
<img width="675" alt="Screenshot 2020-01-27 at 16 57 36" src="https://user-images.githubusercontent.com/1187242/73195795-665aa280-4126-11ea-8888-41fb494c3d58.png">

Chrome:
![Screenshot 2020-01-27 at 16 58 35](https://user-images.githubusercontent.com/1187242/73195796-665aa280-4126-11ea-88a0-908e58ae954f.png)

Safari:
![Screenshot 2020-01-27 at 16 58 53](https://user-images.githubusercontent.com/1187242/73195798-665aa280-4126-11ea-8dbc-cf650a806d86.png)

**Comment by JanPokorny** [comment]
@JD557 Thanks! Looks like crossing out emoji shouldn't be relied on (although it's funny that in all of the above screenshots, the browser managed to correctly cross out the last bullet point, including the emoji...). Let's stick with ⚠.

**Comment by JanPokorny** [comment]
@Mixaill I got to doing this PR, but then I noticed that many info in the table regarding "not implemented"/"not supported by the platform" is wrong. For example, Humble as a platform has no achievements, but they show as ❌. And it got me thinking... is the distinction "not implemented because it's not possible" and "not implemented but possibly could" really important for the user? What's your opinion?

**Comment by reischlfranz** [comment]
I'd like to second the initial request. I have however a slightly different suggestion for proper use:

- ✅ or ✔️ (either is fine) for implemented is fine
- ⚠ for "works with issues" is fine
- ⬛ for "unknown / to be added" is fine, but should be relabeled to "Insufficient information"
- ⬜ should be used for "to be done", since it looks like an unticked box just waiting to be ticked.
- ❌ should be used for "not possible" / "not supported"

Basically the main idea is to swap ⬜ and ❌.

**Comment by JanPokorny** [comment]
@reischlfranz As I found out, this info is hard to gather and harder to keep updated. From the user point of view, works/works with issues/does not work is the important distinction.

**Comment by reischlfranz** [comment]
@JanPokorny I have no dog in this fight, I just stumbled over this discussion and wanted to offer my input.

**Comment by JanPokorny** [comment]
@reischlfranz Understood, thanks for the input. Please don't see this as a fight, it's just a mildly severe case of [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding).

**Comment by UncleGoogle** [comment]
Lets play.


Version name | Vote with
------------------------------------------------- | --:
[✔️❌**ⁿ/ₐ** JanPokorny](https://github.com/UncleGoogle/awesome-gog-galaxy/tree/icons_pokorny)                                             |  👀 
[✅⬜ⁿ/ₐ Custom](https://github.com/UncleGoogle/awesome-gog-galaxy/tree/icons_na) | 🚀 
[✅⬜❌reischlfranz](https://github.com/UncleGoogle/awesome-gog-galaxy/tree/icons_reischlfranz) | ❤️ 
[✔️⬜❌reischlfranz](https://github.com/UncleGoogle/awesome-gog-galaxy/tree/icons_reischlfranz_2) | 🎉

**Comment by Mixaill** [comment]
Looks like v4 will be the winner
